python/create_mask.py

```diff
@@ -0,0 +1,150 @@
+import os.path as op
+
+#=====Check python version=====
+
+print("PYTHON: Starting to create CH masks.")
+if sys.version[0] != '3':
+    raise NameError('=====   Python version 3 must be used for create_mask.py   =====')
+
+#=====Establish pathing=====
+path = ''
+if len(sys.argv) > 1:
+    path = sys.argv[1]
+    exis = op.exists(op.join(path,'meta/'))
+    if exis == 0:
+        raise NameError('===== Path to CH location file does not exist =====')
+
+#=====Find CH location file=====
+print("PYTHON: Finding CHIMERA files.")
+f = glob.glob(op.join(path,"meta/*ch_location*.txt"))
+
+#=====Establish variables and arrays=====
+siz = 4096
+hsiz=siz/2
+iarr = np.zeros((siz, siz) , dtype=np.byte)
+slate, circ = np.array(iarr), np.array(iarr)
+x, y = np.array([]), np.array([])
+fill = 0
+
+#=====Cycle through lines in file=====
+print("PYTHON: Reading files ...")
+with open(f[0],"r") as file:
+
+    print("PYTHON: File opened.")
+
+    for line in file:
+        if line[0:3] != 'Dat':
+
+            #=====Establish solar radius=====
+            if line[0:3] == 'Sol':
+                r = line[21:-1]
+
+            #=====Fill CH boundary locations=====
+            elif line[0:3] == '   ':
+                x,y = np.append(x, int(float(line[6:13]))), \
+                np.append(y, int(float(line[20:27])))
+
+            #=====Conversion factor for pixels to arcsec=====
+            elif line[0:3] == 'Pix':
+                pix2arc = float(line[22:-1])
+
+            else:
+                #=====Fill CH boundaries=====
+                if x.any():
+                    mahotas.polygon.fill_polygon(np.array(list(zip(y.astype(int),x.astype(int)))), \
+                    slate)
+                    iarr[np.where(slate == 1)] = fill
+                    slate[:] = 0	
+                    x, y = np.array([]), np.array([])
+
+                #=====Find CH ID=====
+                if line[0:3] == 'ID:':
+                    id = int(float(line[10:-1]))
+
+                    if id >= 1:
+                        fill = 1
+
+                    else:
+                        fill = 0
+			
+        else:
+            dat = line[13:-1]
+
+
+
+#=====Create datetime object=====
+date=datetime.strptime(dat, '%Y%m%d_%H%M%S')
+
+#=====Create arcsecond meshgrid=====
+xgrid, ygrid = np.meshgrid((np.arange(siz)-hsiz)*pix2arc , (np.arange(siz)-hsiz)*pix2arc)
+center = np.array([0,0])
+
+chs = np.where(iarr > 0)
+slate[chs] = 1
+slate = np.array(slate,dtype=np.uint8)
+
+#=====Create mask of solar disk=====
+print("PYTHON: Creating mask.")
+w = np.where((xgrid-center[0])**2+(ygrid-center[1])**2 <= (float(r)*pix2arc)**2)
+circ[w] = 1.0
+
+#=====Establish plots=====
+plt.figure(figsize = (10, 10))
+
+plt.xlim(-hsiz*pix2arc, hsiz*pix2arc)
+plt.ylim(-hsiz*pix2arc, hsiz*pix2arc)
+
+#=====Plot transparent CH fill=====
+plt.scatter((chs[1]-hsiz)*pix2arc, (chs[0]-hsiz)*pix2arc, marker = 's', \
+s = 0.0183, c = 'black', cmap = 'viridis', edgecolor = 'none', alpha = 0.2)
```

Comment:
You don't need the `\`, as you are inside of a `()`. Then the following line could be indented.